### PR TITLE
doing feedback from SSO PM

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/create-sso-office-add-ins-nodejs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/miscellaneous-sso-updates/docs/develop/create-sso-office-add-ins-nodejs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201811132147405918-556%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=5Ld7tZ1Xm3OEhr2oCfiSozGUPk4koJkbTsqHzHPNU3M%3D&st=2018-11-13T21%3A44%3A00Z&se=2018-12-14T21%3A49%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/create-sso-office-add-ins-nodejs.md [hunk 1]
```diff
@@ -551,7 +552,9 @@ There are two server-side files that need to be modified.
     * The MSGraphHelper class is defined in src\msgraph-helper.ts. 
     * We minimize the data that must be returned by specifying that we only want the name property and only the first 3 items.
 
-    `const graphData = await MSGraphHelper.getGraphData(graphToken, "/me/drive/root/children", "?$select=name&$top=3");`
+    ```javascript
+    const graphData = await MSGraphHelper.getGraphData(graphToken, "/me/drive/root/children", "?$select=name&$top=3");
```
Comment (AlexJerabek) [inline docs/develop/create-sso-office-add-ins-nodejs.md]: With `const` and `await`, isn't this TypeScript?

File: docs/develop/create-sso-office-add-ins-nodejs.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ This article walks you through the process of enabling single sign-on (SSO) in a
     > * The **Before** folder is a starter project. The UI and other aspects of the add-in that are not directly connected to SSO or authorization are already done. Later sections of this article walk you through the process of completing it. 
     > * The **Completed** version of the sample is just like the add-in that you would have if you completed the procedures of this article, except that the completed project has code comments that would be redundant with the text of this article. To use the completed version, just follow the instructions in this article, but replace "Before" with "Completed" and skip the sections **Code the client side** and **Code the server** side.
     > * The **Completed Multitenant** version is a completed sample that supports multitenancy. Explore this sample if you intend to support Microsoft accounts from different domains with SSO.
+    > *Regardless of which version you use, you will need to trust a cert for the localhost. See the "IMPORTANT" note in the Readme of the repo.*
```
Comment (AlexJerabek) [inline docs/develop/create-sso-office-add-ins-nodejs.md]: Nit: "certificate" instead of "cert"
(I'm not actually sure...)

Comment (ElizabethSamuel-MSFT) [inline docs/develop/create-sso-office-add-ins-nodejs.md]: Is this meant to be another bullet point or an extension of the 'Completed Multitenant' item? Current behavior is that it's an italicized sentence in the 'Completed Multitenant' bullet point.

File: docs/develop/create-sso-office-add-ins-nodejs.md (hunk 1 above)
Comment (Rick-Kirkham) [reply docs/develop/create-sso-office-add-ins-nodejs.md]: Yes. This is a TypeScript walkthrough.

Comment (AlexJerabek) [reply docs/develop/create-sso-office-add-ins-nodejs.md]: Then the code type should be "typescript" instead of "javascript" :-)

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/create-sso-office-add-ins-nodejs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/miscellaneous-sso-updates/docs/develop/create-sso-office-add-ins-nodejs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201811132304530922-556%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=5DedRjlzdEerXvD2gMCAxHfoE4hK%2B%2BlBDh%2B0VOPHhWw%3D&st=2018-11-13T23%3A01%3A05Z&se=2018-12-14T23%3A06%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/create-sso-office-add-ins-nodejs.md (hunk 2 above)
Comment (Rick-Kirkham) [reply docs/develop/create-sso-office-add-ins-nodejs.md]: It's meant to be a completely separate paragraph _after_ the bulleted list. Fixed now.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/create-sso-office-add-ins-nodejs.md](https://github.com/OfficeDev/office-js-docs-pr/blob/miscellaneous-sso-updates/docs/develop/create-sso-office-add-ins-nodejs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201811132316369588-556%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=yKz1C7jcMzkr%2FbqpWxlrIv64ofX6trh%2ByFCDmxgiFcU%3D&st=2018-11-13T23%3A12%3A55Z&se=2018-12-14T23%3A17%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**